netket/vqs/mc_state.py

```diff
@@ -484,9 +541,31 @@ def __str__(self):
         )
 
 
-# serialization
+# overloads for batched expect
+expect.register()
+def expect_nominibatch(vstate: MCState, operator: AbstractOperator, minibatch_size: None):
+    return expect(vstate, operator) 
+
+expect.register()
+def expect_minibatch(vstate: MCState, operator: AbstractOperator, minibatch_size: int):
+    σ = vstate.samples
+
+    n_chains = σ.size[1]
+    minibatch_length = minibatch_size // n_chains
+
+    σr = σ.reshape(-1, minibatch_length, σ.shape[1], σ.shape[2])
 
+    i = 0
+    stats = []
+    while i < σr.shape[0]
+        vstate._samples = σr[i]
+        stats.append(expect(vstate, operator))
 
+    # merge
+    return merged_stats
```

Comment:
where is ```merged_stats``` defined?

Reply:
it's not.
I'm toying with two ways to do it